docs/source/using/con_static-ip-overview.adoc

```diff
@@ -0,0 +1,7 @@
+Most hypervisors do not support extending the lease time when the IP is assigned using DHCP.
+This might lead to a new IP being assigned to the VM after a restart as it will conflict with the security certificates generated for the old IP.
+This will make {project} completely unusable until a new instance is set up by running `minishift delete` followed by `minishift start`.
+
+To prevent this, {project} includes the functionality to set a static IP address to the VM.
+This will prevent the IP address from changing between restarts.
```

Comment:
sounds like duplicate use of `prevent` in a too short timeframe. `This will stop the IP address from ...` although harsh(er) in tone, it is clear in what it does.